modules/web-console/docker/compose/docker-compose.yml

```diff
@@ -30,10 +30,6 @@ services:
     # Restart on crash.
     restart: always
     environment:
-      # Port for serving frontend API
-      - server_port=3000
```

Comment:
I see 3 files with string "- server_host=0.0.0.0" in 8.7-master:
ggprivate\modules\web-console\docker\cloud\docker-compose.yml
ggprivate\modules\web-console\docker\console-on-premise\docker-compose.yml
incubator-ignite\modules\web-console\docker\compose\docker-compose.yml - this one is fixed by you.
What about files in ggprivate?

Reply:
fixed